plugins/core/Standard/qRANSAC_SD/qRANSAC_SD_Commands.h

```diff
@@ -454,6 +449,12 @@ struct CommandRANSAC : public ccCommandLineInterface::Command
 					}
 				}
 			}
+			if (cmd.autoSaveMode())
+			{
+				QString errorStr = cmd.exportEntity(clCloud); // The original cloud may have had normals added
+				if (!errorStr.isEmpty())
+					cmd.warning(errorStr);
```

Comment:
Style: Add "{" and "}" around single statements in an `if`

Reply:
I've added { and } throughout the entire file now for consistency